deployments/k8s/configmap.yaml

```diff
@@ -46,6 +46,14 @@ data:
     - type: kubelet-stats
```

Comment:
looks like we've also got `docker-container-stats` monitors enabled. do we it when we've . got `kubelet-stats`monitor enabled?

Reply:
I need to make that a flag so people not using Docker can easily turn it off.

Reply:
I checked and this is already there.